shared/common/restful-authentication/token-method.mdx

```diff
@@ -4,4 +4,17 @@
 
     -   The Signaling token generated from your server
 
-    -   The Signaling user ID used to generate the Signaling token
\ No newline at end of file
+    -   The Signaling user ID used to generate the Signaling token
+
+- **AccessToken2**
+
+    You need to fill the `Authorization: agora token=` field with the RTM [AccessToken2](../develop/authentication) generated from your server.
+
+- **AccessToken**
+
+    You need to fill the `x-agora-token` and `x-agora-uid` fields in the request header with the following information:
+
+    - The RTM [AccessToken]((https://docs.agora.io/en/Real-time-Messaging/token_server_rtm?platform=All%20Platforms)) generated from your server
```

Comment:
The sample code in RESTful auth FAQ is for Token 006. Where's the stuff for Token 007?
<img width="1905" alt="image" src="https://user-images.githubusercontent.com/101544891/192252786-d0556172-bad1-407e-86ca-653520cd84f7.png">

Reply:
Can you supply the code please and I will update

Reply:
https://github.com/AgoraIO/Docs-Source/issues/409
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/101544891/192502413-0cf27828-4ce0-4c5f-9d6a-093c0abecf94.png">